src/daemon/daemon.cpp

```diff
@@ -1053,6 +1053,15 @@ mp::Daemon::Daemon(std::unique_ptr<const DaemonConfig> the_config)
         }
     });
     source_images_maintenance_task.start(config->image_refresh_timer);
+
+    try
+    {
+        config->factory->hypervisor_health_check();
+    }
+    catch (const std::runtime_error& e)
+    {
+        mpl::log(mpl::Level::warning, category, fmt::format("Hypervisor health check failed: {}", e.what()));
```

Comment:
Should this be warning, or error even?

Reply:
Well, it is `warning` :slightly_smiling_face:

I refrained from `error` since it's not really fatal since `hypervisor_health_check()'` will run again when an instance starts/launches and that is surfaced to the client in order for the user to take action.  It's probably splitting hairs between those two log severity values though...

Reply:
Or, after re-reading what you said, perhaps you mean it should be a lesser severity like `info` or something?

Reply:
No no, I did mean warning or error, I just somehow looked for `mpl::warning()` ;P

Reply:
lol, ok